runtime/src/main/java/com/flipkart/flux/guice/module/ContainerModule.java

```diff
@@ -148,14 +145,20 @@ Server getDashboardJettyServer(@Named("Dashboard.service.port") int port,
 	@Singleton
 	Server getAPIJettyServer(@Named("Api.service.port") int port,
 							 @Named("APIResourceConfig")ResourceConfig resourceConfig,
+							 @Named("Api.service.acceptors") int acceptorThreads,
```

Comment:
change the values in configuration.yml to jetty default

  service.acceptors: 4
  service.selectors: 4
  service.workers: 200